api/graphql/schema.graphql

```diff
@@ -1609,6 +1620,10 @@ input RoleAssignmentHasMany {
   detach: RolesInput
   sync: RolesInput
 }
+input UpdateApplicationInput {
+  # can only connect a step, not disconnect or sync
+  connectSubmittedStep: ApplicationStep
```

Comment:
Yeah, since this is just a flat array on enums I might go with appendSubmittedStep.  I think trying to use Laravel relationship terminology is just going to further confuse.

Reply:
[switch to insertSubmittedStep](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6114/commits/1ba7a1b322b1d8a2e4aaa81d5e06f69bb5c39a98)